mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/model/dependency/PojoOtherEntityIndexingDependencyConfigurationContext.java

```diff
@@ -23,9 +21,8 @@
 	 * @return {@code this}, for method chaining.
 	 * @throws org.hibernate.search.util.common.SearchException If the given path cannot be applied to the entity type.
 	 * @see #use(PojoModelPathValueNode)
-	 * @hsearch.experimental This feature is under active development.
-	 *    Usual compatibility policies do not apply: incompatible changes may be introduced in any future release.
 	 */
+	@Incubating
```

Comment:
Looks like you forgot the method just below.

Reply:
Yes i did. Fixed